msgs-circe/src/main/scala/com/thenewmotion/ocpi/msgs/circe/v2_1/CommonJsonProtocol.scala

```diff
@@ -76,12 +76,21 @@ trait CommonJsonProtocol {
         SuccessResp(statusCode, statusMsg, timestamp)
     }
 
+  def successPagedRespWithoutDataD[T]: Decoder[SuccessResp[Iterable[T]]] =
+    implicitly[Decoder[SuccessResp[Unit]]].map(_.copy(data = Iterable.empty[T]))
+
+  def successPagedRespWithData[T: Decoder]: Decoder[SuccessResp[Iterable[T]]] = successRespD[Iterable[T]]
+
+  def successPagedResp[T: Decoder]: Decoder[SuccessResp[Iterable[T]]] =
+    successPagedRespWithData[T].handleErrorWith(_ => successPagedRespWithoutDataD[T])
```

Comment:
What happens when there is an error in one of the elements of the data-array? Does it then silently turn data into an empty array, instead of rejecting the json?